services/web/src/admin/App.js

```diff
@@ -0,0 +1,52 @@
+import 'semantic-ui-less/semantic.less';
+import { hot } from 'react-hot-loader/root';
+
+import { Switch, Route, Redirect } from 'react-router-dom';
+import React from 'react';
+
+import AuthSwitchRoute from './routes/AuthSwitch';
+import Protected from './routes/Protected';
+
+import Homepage from './screens/Homepage';
+import Dashboard from './screens/Dashboard';
+import Settings from './screens/Settings';
+import Invites from './screens/Invites';
+import Users from './screens/Users';
+import AcceptInvite from './screens/Auth/AcceptInvite';
+import Login from './screens/Auth/Login';
+import Shops from './screens/Shops';
+import Shop from './screens/Shop';
+import Logout from './screens/Auth/Logout';
+import ForgotPassword from './screens/Auth/ForgotPassword';
+import Signup from './screens/Auth/Signup';
+import ResetPassword from './screens/Auth/ResetPassword';
+
+const App = () => (
+  <Switch>
+    <AuthSwitchRoute exact path="/admin/" loggedIn={Dashboard} loggedOut={Homepage} />
+    <Protected exact path="/admin/shops" component={Shops} />
```

Comment:
most this stuff was meant as user specify not admin specify. 
I suspect that needs to be moved a bit around

Reply:
I agree, but recall a number of admin specific things in `AuthSwitchRoute` and `Protected` that I didn't want to mess with so kept them admin-specific for now rather than go in and mess around.

Reply:
Abandoning the directory updates now so none of this applies anymore. "admin" vs "public" will be something that you'll have to set up yourself if/when you need it going forward (however there are helpers that will make that easier).